eng/config/test/Core/app.config

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-
-<!-- The default app.config for desktop portable test projects that do not specify one -->
-<configuration>
-  <system.diagnostics>
-    <trace>
-      <listeners>
-        <remove name="Default" />
-        <add name="ThrowingTraceListener" type="Microsoft.CodeAnalysis.ThrowingTraceListener, Roslyn.Test.Utilities" />
```

Comment:
At one point this file was used to ensure all of the `exe` that we build in our infra use a throwing trace listener vs. the default ones. How does that work now for our exe files?

Reply:
It never worked. .NET Core does not use this file.

Reply:
Yes but this is important for .NET Desktop too for the same reasons.

Reply:
The .NET Desktop version of this file has not been changed.
https://github.com/dotnet/roslyn/blob/master/eng/config/test/Desktop/app.config